devops/scripts/bootstrap_azure_env.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+# Uncomment this line to debug (will echo secrets!)
+# set -o xtrace
+
+FLAGS=()
+if [[ "$#" -gt 0 ]]; then
+  RAW_ARGUMENTS="${*//,/ }"
+  # shellcheck disable=SC2206
+  FLAGS=(${RAW_ARGUMENTS})
+fi
+
+has_flag() {
+  local desired="$1"
+  for flag in "${FLAGS[@]}"; do
+    if [[ "${flag}" == "${desired}" ]]; then
+      return 0
+    fi
+  done
+  return 1
+}
```

Comment:
Are you using this?

Reply:
Good catch. I removed the flags. Will get rid.

Reply:
Done